src/modules/fw_att_control/fw_att_control_main.cpp

```diff
@@ -748,12 +740,13 @@ FixedwingAttitudeControl::task_main()
 	vehicle_status_poll();
 	vehicle_land_detected_poll();
 	battery_status_poll();
+	_sub_airspeed.update();
```

Comment:
@dagar Why don't we follow the same pattern here as for the topics? e.g `battery_status_poll();` etc.